### DISCUSSION TITLE
How can  i get pulled records

### DISCUSSION THREAD

**Discussion by fzwu** [goharbor/harbor#17993] [Q&A] [answered]
Hi:
I use the Harbor API and need to get the record of artifact tags pulled，Which interface can I get it through

**Comment by AllForNothing** [comment]
@chlins Can you help with this?

**Comment by chlins** [comment, marked as answer]
What's the meaning of pulled records? the pulling counts for tag?

**Reply by fzwu** [reply]
Not the pulling counts for tag,including pulled time, pulled user, like the logs of artifact

**Reply by chlins** [reply]
For the pull time, you can call the artifact API which includes that in the response, and the user who pull the artifact needs to call audit log API to retrieve.

**Reply by chlins** [reply]
You can go to the API center from harbor UI to explore harbor APIs.
